### ISSUE TITLE
Volta error: 'volta-shim' should not be called directly.

### ISSUE THREAD

**Issue by lunaluna** [volta-cli/volta#1439]
Hi!

I used Volta 1.1.1 installed from homebrew, and today I used the command `brew upgrade`, so Volta updated automatically to `1.1.1_1`.
Then I used the command `volta setup`, and `npm run watch` as always.
But terminal displayed `Volta error: 'volta-shim' should not be called directly.`

Volta worked without any problems at version 1.1.1, so I searched your documents, and re-install version 1.1.1.

https://docs.volta.sh/advanced/installers#installing-old-versions

```
curl https://raw.githubusercontent.com/volta-cli/volta/8f2074f423c65405dfba9858d9bcf393c38ffb45/dev/unix/volta-install.sh | bash -s -- --version 1.1.1
```

I use MBP M1 MAX 2021 and macOS Ventura 13.2.

Is it correct way to recover? And how it works at version 1.1.1_1? 

Thank you.

**Comment by lunaluna** [comment]
It works at `1.1.1` now, and it has no problem.

**Comment by ryo0301** [comment]
Instructions for installing volta1.1.0 with Homebrew
```
$ brew tap-new $USER/volta
$ brew extract --version=1.1.0 volta $USER/volta
$ brew install $USER/volta/volta@1.1.0
$ brew developer off
```

**Comment by rmichalak** [comment]
I believe it's fixed with `1.1.1_2` in homebrew: https://github.com/Homebrew/homebrew-core/pull/122350

**Comment by lunaluna** [comment]
Thank you @ryo0301 @rmichalak !
Now I know volta `1.1.1_2` works!

I'll close this issue.
Thank you for your help!

But now... I got same probrem as #1430... (´・ω・｀)
